### PR TITLE
Include the Catalyst vs ASF roles distinction in the Catalyst Program

### PR DESCRIPTION
**Pull request by michaelsembwever**
```
 patch by Mick Semb Wever; reviewed by Dinesh Joshi, Paulo Motta
```

Lots of trailing whitespace removal here.  See comment for main change.

### CODE REVIEW

File: site-content/source/modules/ROOT/pages/cassandra-catalyst-program.adoc
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/cassandra-catalyst-program.adoc:132]: main change.

**Comment by michaelsembwever** [comment]
@dineshjoshi @pauloricardomg , waiting on review from you both.

**Review by pauloricardomg — APPROVED** [review]

**Comment by pauloricardomg** [comment]
👍 lgtm

**Review by dineshjoshi — APPROVED** [review]
+1, thanks!

**Pull request merged**
